core/contract/src/main/java/org/eclipse/dataspaceconnector/contract/policy/PolicyEngineImpl.java

```diff
@@ -0,0 +1,135 @@
+/*
+ *  Copyright (c) 2021 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+package org.eclipse.dataspaceconnector.contract.policy;
```

Comment:
```PoliceEngine``` now lives in ```spi```, ```PolicyEngineImpl``` in ```contract```. What will happen with the ```policy-engine``` module?

Reply:
Oh shoot, I forgot about that name. I think it should be changed to policy-evaluator. We also need to adjust the package name. I think this should be done in a separate PR. Do you like the name policy-evaluator?